### ISSUE TITLE
[Bug]: cannot load db.json sometime

### ISSUE THREAD

**Issue by darkjackx** [massCodeIO/massCode#488]
### Describe the bug

Environment:
PC1 load snippets by mounting google drive.

Now, PC1 show 0 snippets in setting after start up the program, 
Its not working even click Reload Storage / Load Existing Storage.
Then i copy the db.json to local drive and load still not working.

But i using another PC2/Mac3 they can load the db with same version of masscode

I can send you the db.json file by email, because it have many snippets and maybe contain password.
So i don't to send here for public

### To reproduce

1. Go to ...
2. Click on ...
3. Scroll down to ..

### Version

3.11.0

### OS

windows 11

### Validations

- [X] Follow our [Code of Conduct](https://github.com/massCodeIO/massCode/blob/master/CODE_OF_CONDUCT.md)
- [X] Check that there isn't [already an issue](https://github.com/massCodeIO/massCode/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/massCodeIO/massCode/discussions).

**Comment by ondrovic** [comment]
might be related to this: [481](https://github.com/massCodeIO/massCode/issues/481)

**Comment by snowraincloud** [comment]
windwos modifies reserved ports :
netsh int ipv4 set dynamic tcp start=49152 num=16384
net stop winnat
net start winnat

**Comment by ondrovic** [comment]
Yep the best thing to do would be to make the API_PORT definable in preferences there is a discussion item about it here
https://github.com/massCodeIO/massCode/discussions/329 , I think that is going to be the only way to resolve these since windows does what it wants 🤣

**Comment by antonreshetov** [comment]
**Closing as outdated.**

This issue was reported against an older version of massCode (v3/v4) that relied on the legacy storage system (`db.json` / SQLite). massCode v5 introduces a completely new architecture based on a markdown vault, and we no longer maintain or fix issues for versions below v5.

If you can still reproduce this on the latest v5 release, please open a new issue with up-to-date reproduction steps. Thanks for your report and for using massCode!
